packages/site/src/data/descriptions.ts

```diff
@@ -0,0 +1,8 @@
+export const descriptions = {
+    "angular": "",
```

Comment:
Maybe we need to coordinate/communicate this change to the maintainers of the other pods?

Reply:
Yeah, I was hoping we don't have `undefined` as the value. Yeah we should communicate with other pod maintainers @dustinsgoodman @Zelig880